### PR TITLE
DDF-3828 Fixes the css for the input with param component

### PR DESCRIPTION
**Pull request by andrewkfiedler**
#### What does this PR do?
- Fixes the css for the input with param component.
- Previously it would spill outside it's container, now it sizes it's children correctly.

#### Who is reviewing it? 
@shaundmorris 
@bdeining 
@djblue 
@adimka 
@mackncheesiest 
@andrewzimmer

#### How should this be tested?
Test out the near input in the advanced search.

#### What are the relevant tickets?
[DDF-3828](https://codice.atlassian.net/browse/ddf-3828)

[Video](https://www.youtube.com/watch?v=cg3Gh6DVF5k&feature=youtu.be)

### CODE REVIEW

File: ui/packages/catalog-ui-search/src/main/webapp/dev/component/input-guide/input-guide.view.js [hunk 1]
```diff
@@ -177,7 +177,7 @@ module.exports = BaseGuideView.extend({
                     distance: 10
                 }],
                 type: 'NEAR',
-                param: 'within',
+                param: 'within a few feet of',
```
Comment (mackncheesiest) [inline ui/packages/catalog-ui-search/src/main/webapp/dev/component/input-guide/input-guide.view.js]: ❗️ We don't want this change

Comment (andrewkfiedler) [reply ui/packages/catalog-ui-search/src/main/webapp/dev/component/input-guide/input-guide.view.js]: Why not? 😂

**Comment by andrewkfiedler** [comment]
build now

**Comment by andrewkfiedler** [comment]
build now

**Comment by andrewkfiedler** [comment]
build now

**Pull request merged**